spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AppRegistryController.java

```diff
@@ -116,7 +117,8 @@ public AppRegistryController(AppRegistry appRegistry,
 			}
 		}
 		Collections.sort(list);
-		return pagedResourcesAssembler.toResource(new PageImpl<>(list), assembler);
+		return pagedResourcesAssembler
+				.toResource(new PageImpl<>(list, pageable, appRegistry.findAll().size()), assembler);
```

Comment:
The `appRegistry.findAll()` can be reused from what we computed above in the same method.